features/logging/collector.feature

```diff
@@ -9,6 +9,17 @@ Feature: collector related tests
   Scenario: All nodes logs are sent to Elasticsearch
     Given the master version == "4.1"
     Given evaluation of `cluster_logging('instance').fluentd_ready_pods.map(&:ip)` is stored in the :collector_pod_ips clipboard
+    #A workaround to https://bugzilla.redhat.com/show_bug.cgi?id=1776594
+    And I wait for the ".operations" index to appear in the ES pod with labels "es-node-master=true"
+    Given I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | .operations.* |
+      | op           | DELETE                                                                                                                                                                                                                     |
```

Comment:
missing | in the end.

Reply:
Stranger. There is '|' is in the end.   try to remove the spaces between DELETE and |.